app.py

```diff
@@ -26,19 +26,49 @@ async def startup():
     loop.create_task(client.start(TOKEN))
 
 
-@server.get('/users/{user_id}/roles')
+@server.get("/users/{user_id}/roles")
 async def test_endpoint(user_id: int) -> dict:
     """Get a user's roles."""
     guild = client.get_guild(GUILD_ID)
     try:
         member = await guild.fetch_member(user_id)
     except discord.NotFound:
-        raise fastapi.HTTPException(status_code=404, detail='User not found.')
+        raise fastapi.HTTPException(status_code=404, detail="User not found.")
     return {
-        'roles': [
+        "roles": [
+            {"name": role.name, "id": role.id}
+            for role in member.roles
+            if not role.is_default()
+        ]
+    }
+
+
+@server.get("/bans")
+async def ban_endpoint():
+    """Get a guilds bans."""
+    guild = client.get_guild(GUILD_ID)
+    try:
+        ban_list = []
+        bans = await guild.bans()
+        for ban in bans:
+            ban_list.append(ban)
```

Comment:
```suggestion
        ban_list = await guild.bans()
```
[According to the docs](https://discordpy.readthedocs.io/en/latest/api.html#discord.Guild.bans), this function returns a list anyway.